lore/technology/drone_manufacturer.mediawiki

```diff
@@ -0,0 +1,16 @@
+=History=
+Due to advancements in the field of macro-scale 3D printing after the field of nanotechnology was largely abandoned due to numeous scandals, manufacturing delicate, yet strong metal drone casings could be done reliably in an automated assembly process. Using this as a starting point, NanoTrasen decided to devote a substantial portion of their research and development facilities to achieving the goal of fully-automated maintenance and construction robots by 2520.
```

Comment:
"... due to nume**r**ous scandals..."

Reply:
I was trying to reference the Grey Tide from Polaris' lore, but I forgot the details surrounding it and didn't know if it was canon anyway.

Reply:
I'm pointing out a typo.